packages/web/src/pages/collection-page/components/desktop/CollectionPage.tsx

```diff
@@ -71,6 +75,16 @@ const EmptyPage = (props: EmptyPageProps) => {
   )
 }
 
+const NoSearchResultsPage = () => {
```

Comment:
nit: page seems like an odd name, but I see we use `EmptyPage` as well.

Reply:
You are not wrong. I was copying the existing empty content without thinking about the naming. Will update.